src/app/app.config.ts

```diff
@@ -0,0 +1,10 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+import { provideHttpClient } from '@angular/common/http';
+
+import { routes } from './app.routes';
+import { provideClientHydration } from '@angular/platform-browser';
+
+export const appConfig: ApplicationConfig = {
+  providers: [provideRouter(routes), provideClientHydration(), provideHttpClient()]
```

Comment:
I'm not sure if you're need provideClientHydration :)

Reply:
This was also there already, so I didn't delete it. You think that Angular is playing with me? ;)